image-layout.md

```diff
@@ -2,7 +2,7 @@
 
 The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
 This layout MAY be used in a variety of different transport mechanisms: archive formats (e.g. tar, zip), shared filesystem environments (e.g. nfs), or networked file fetching (e.g. http, ftp, rsync).
-Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appropriate manifest from the manifest list, unpacking the filesystem serializations in the correct order, and then converting the image configuration into an OCI Runtime config.json.
+Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appropriate manifest from the manifest list, unpacking the [filesystem layers](./layer.md) in the correct order, and then converting the image configuration into an OCI Runtime config.json.
```

Comment:
On Tue, Sep 13, 2016 at 02:12:47PM -0700, Vincent Batts wrote:
> > -Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appropriate manifest from the manifest list, unpacking the filesystem serializations in the correct order, and then converting the image configuration into an OCI Runtime config.json.
> > +Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appropriate manifest from the manifest list, unpacking the [filesystem layers](./layer.md) in the correct order, and then converting the image configuration into an OCI Runtime config.json.
>
> we've been through this. This is for explicit relative path. It's
> not a rule, but many tools rely on such to determine that it is a
> relative path.

Can you point to such a tool?  If we want to require explicit ./,
we'll want a new PR to update many existing links.